index.html

```diff
@@ -196,14 +127,16 @@ <h2>Principles</h2>
 <a data-cite="design-principles#priority-of-constituencies">priority of constituencies</a>.
 We will build new web technologies in a collaborative matter
 according to open processes
-(for example, the <a href="https://www.w3.org/Consortium/Process/">W3C process</a>).
+(for example, the [W3C process](https://www.w3.org/Consortium/Process/)
```

Comment:
This wound up broken at https://w3ctag.github.io/ethical-web-principles/#noharm.

Reply:
you are right. Having a look.

Reply:
Oh right, I forgot! GitHub flavored (maybe all?) markdown doesn't work inside a html tag. In this case, a `<dd>`.

Reply:
https://github.com/w3ctag/ethical-web-principles/pull/43